module/x/gravity/keeper/pool.go

```diff
@@ -88,17 +81,10 @@ func (k Keeper) cancelSendToEthereum(ctx sdk.Context, id uint64, s string) error
 		return fmt.Errorf("can't cancel a message you didn't send")
 	}
 
-	isCosmosOriginated, denom := k.ERC20ToDenomLookup(ctx, common.HexToAddress(send.Erc20Token.Contract))
+	_, denom := k.ERC20ToDenomLookup(ctx, common.HexToAddress(send.Erc20Token.Contract))
 	amountToRefund := send.Erc20Token.Amount.Add(send.Erc20Fee.Amount)
 	coinsToRefund := sdk.NewCoins(sdk.NewCoin(denom, amountToRefund))
 
-	// If it is not cosmos-originated the coins are minted
-	if !isCosmosOriginated {
-		if err := k.bankKeeper.MintCoins(ctx, types.ModuleName, coinsToRefund); err != nil {
-			return sdkerrors.Wrapf(err, "mint vouchers coins: %s", coinsToRefund)
-		}
-	}
-
 	if err := k.bankKeeper.SendCoinsFromModuleToAccount(ctx, types.ModuleName, sender, coinsToRefund); err != nil {
```

Comment:
this isn't done anymore?

Reply:
at least I don't see in batch.go?

Reply:
only unbatched transactions can be cancelled, that is why there is no logic change in batch.go

Reply:
as we don't burn them when there are inserted in the tx pool, we don't need to mint them anymore when we cancel the tx